packages/web/stencil.config.ts

```diff
@@ -51,5 +51,6 @@ export const config: Config = {
   validatePrimaryPackageOutputTarget: true,
   extras: {
     enableImportInjection: true,
+    experimentalSlotFixes: true
```

Comment:
Is there a way for us to guard for things like these that are experimental?

Reply:
This addition doesn't directly fix the issue this PR addresses. I just decided to add it into our build to hopefully protect against any slot issues.